### DISCUSSION TITLE
Companion 2.8 Radio Simulation

### DISCUSSION THREAD

**Discussion by tommp100** [EdgeTX/edgetx#2937] [Q&A]
When I try to run the Radio Simulation in Companion 2.8 I get an error message "Storage Warning - Unable to read valid radio settings". The debug file shows two error lines:
[W] Translations::tryLoadTranslator():165 - Error installing translation file "qt" for: "en_US" from: ":/translations"
[W] Translations::tryLoadTranslator():165 - Error installing translation file "qtbase" for: "en_US" from: ":/translations"

I did some searches on qt and qtbase but did not find anything I could apply to Companion.  Would appreciate any troubleshooting advice from anyone. 
I tried uploading the Companion  .log file but get an error message saying "We don't support that file type"

**Comment by elecpower** [comment]
Change file extension to txt and it should allow.
Is this the release or selfbuild version?
Operating system?
Are you launching the simulator from within Companion or the standalone Simulator?

**Reply by tommp100** [reply]
[CompanionDebug_22-12-23_13-11-48.txt](https://github.com/EdgeTX/edgetx/files/10321000/CompanionDebug_22-12-23_13-11-48.txt)
This is the released version "companion-windows-2.8.0.exe"
The operating system is windows 64
I am attempting to launch directly from within Companion
